src/user.py

```diff
@@ -6,7 +6,7 @@
 import json
 
 class CreateUser(Resource):
-	def post(self):
+	def put(self):
```

Comment:
Creating a resource (user) is a `POST` request

Reply:
@aaron2212 My comments are as above.